### ISSUE TITLE
"Import without source" deprecated?

### ISSUE THREAD

**Issue by stmichael** [stmichael/data-import#42]
We have an acceptance test "import without source". The import config looks like this:

``` ruby
import 'Static objects' do
  to 'objects'

  after do
    target_database[:objects].insert(:name => 'a box')
    target_database[:objects].insert(:name => 'my cat')
  end
end
```

I think this is deprecated since we now have script blocks. You could easily rewrite this definition to:

``` ruby
script 'Static objects' do
  body do
    target_database[:objects].insert(:name => 'a box')
    target_database[:objects].insert(:name => 'my cat')
  end
end
```

This is even less code.

@senny do you agree with me?

**Comment by senny** [comment]
I agree.

**Comment by senny** [comment]
I think we could even go one step further and drop the functionality for "#after" blocks. In the end, you can just write a script-block with a dependency on the import and you get the same benefits.

**Comment by stmichael** [comment]
Yes, master Yoda ;-)

**Comment by senny** [comment]
let's deprecate the after block then and have the message say you should use a script block with a dependency.

**Comment by senny** [comment]
closed in favor of #45
